### DISCUSSION TITLE
choose/change font?

### DISCUSSION THREAD

**Discussion by viper73** [sumatrapdfreader/sumatrapdf#2433] [Ideas]
Hello,

i really like this epub reader (3.4 preRelease). But i dont like the font of the epub books. I like it better when i can choose the font which is more easy to read for me. In other readers i can change the font to my favourite one. Will this be available in one future release? This would be an huge upgrade of this program.

Or is it possible to change the font somewhere in the program myself?

Greets, Viper73

**Comment by kjk** [comment]
Yes, I plan to allow changing font for epub. This is tracked in https://github.com/sumatrapdfreader/sumatrapdf/issues/2146

**Reply by viper73** [reply]
Any news when this will be available?

**Reply by WangYH195** [reply]
i want to insert some chinese characters as free text, but some words cant display by the defaut font(only 3 choices in V3.5.2 today).
i tried in acrobat to save those words in local font and open the pdf by sumatra. They come back again! So, if i can choose my local font when i insert a free text, everything will be fine.
